src/main/java/Entity/Message.java

```diff
@@ -0,0 +1,76 @@
+package Entity;
+
+public class Message {
```

Comment:
Maybe you could consider a different name for this class that represents the tasks to be stored as I initially thought this meant the input messages instead of the tasks that are being created and stored in the IP.